app/src/main/java/org/oppia/app/player/state/StateFragment.kt

```diff
@@ -7,11 +7,27 @@ import android.view.View
 import android.view.ViewGroup
 import org.oppia.app.fragment.InjectableFragment
 import org.oppia.app.player.audio.CellularDataInterface
+import org.oppia.app.player.exploration.EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY
 import javax.inject.Inject
 
 /** Fragment that represents the current state of an exploration. */
 class StateFragment : InjectableFragment(), CellularDataInterface {
 
+  companion object {
+    /**
+     * Creates a new instance of a StateFragment
+     * @param topicId
+     * @return [StateFragment]: Fragment
+     */
+    fun newInstance(topicId: String): StateFragment {
```

Comment:
Is this topicId actually used anywhere? If not, prefer to remove.

Reply:
the explorationId is retrieved in StateFragmentPresenter https://github.com/oppia/oppia-android/blob/324d493a49b4bd137ba3a028737545dd65d2ebb2/app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt#L58